frontend/src/components/Sidebar/prepareRoutes.ts

```diff
@@ -135,7 +136,7 @@ function prepareRoutes(t: (arg: string) => string) {
       placement = parent['subList'];
     }
 
-    placement.push(item);
+    placement.push(_.cloneDeep(item));
```

Comment:
Perhaps cloneDeep at the top of the loop, and use the clonedItem everywhere?

This way parent will also be cloned.

Reply:
The parent will come from the routes list or the cloned item, so I don't think it will be used without being a new/cloned object. But I have added the clone to the beginning of the item use, as it's safer for future changes to have it there. Thanks.